ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss

```diff
@@ -11,7 +11,7 @@
   height: 32px;
   width: 100%;
 
-  &--selected {
```

Comment:
`--` is used in BEM to modify an existing class because of a temporary state change, so what you had before seems correct to me.

Reply:
I had different review feedbacks about it, that is reason I changed it

Reply:
Ah, no worries then 👍🏻